### DISCUSSION TITLE
Two or more sliders on one page don't work

### DISCUSSION THREAD

**Discussion by tzijay** [datenstrom/community#742] [Report a bug] [answered]
Hi yellow-team, 
i use the slider extension and the recent yellow-code.
Have you ever implemented two or more sliders on one page? When i use this code, the effect is, that i see both silders as well, but when i click on the first one the second is changing the picture. When i click on the second it works fine and only the second is changing pics.
Do you have any hints or is ist just a bug? I already tried differents names and slider settings. Thanks for some help!
```
[slider pics1/]
[slider pics2/]
```

**Comment by schulle4u** [comment]
I did a quick research and found out that in theory the used SplideJS library [supports multiple sliders on the same page](https://splidejs.com/guides/getting-started/#multiple-sliders), at least in the latest version. The slide extension currently uses an older version of the SplideJS library which might not support multiple sliders. Probably the extension itself also would have to implement this function.

**Comment by schulle4u** [comment, marked as answer]
@tzijay The [Slider extension](https://github.com/datenstrom/yellow-extensions/tree/master/source/slider) has just been updated, fixing a bug which prevented loading multiple sliders on the same page. Does this version work for you?

**Reply by tzijay** [reply]
Wow, this was really fast. Thank yor for your quick research and the fast bugfixing! I already tried and it works very well!
